src/common/CustomButton.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import Button from "@material-ui/core/Button";
+
+interface ButtonProps {
+  label: string;
+  className: string;
+  onClick: () => void;
+  variant: "contained" | "outlined" | "text";
+}
+
+interface ButtonState {
+  hover: boolean;
+}
+
+export const CustomButton: React.FunctionComponent<ButtonProps> = (props) => {
+  const [state, setState] = React.useState<ButtonState>({ hover: false });
+
+  const handleMouseEnter = () => {
```

Comment:
Great job Willochs. 

I am proud of your growth.

I will merge this Mr.


Just something to note:

For a small component like this handling hover with react state, works. But if you have a large list, is better to use plain css... it's more performant. Great job